Kaduna_Tim_Arrays/src/Kaduna_Tim_Arrays/KadunaArrays.java

```diff
@@ -0,0 +1,30 @@
+package Kaduna_Tim_Arrays;
+
+/**
+ * Created by Kanini W'Tang on 7/12/2017.
+ */
+public class KadunaArrays {
+
+    public static void main(String[] args) {
+
+        String[] fieldArray = new String[10];
+
```

Comment:
All the arrays you created are not initialized Tim. Kindly go and assign values to them before using it.